template/packages/backend/src/config.service.ts

```diff
@@ -40,11 +43,13 @@ export class ConfigService {
   }
 
   get isDev(): boolean {
-    return this.env.NODE_ENV === "development"
+    return (
+      process.env.NODE_ENV !== "production" && process.env.NODE_ENV !== "test"
```

Comment:
I'm sorry, I haven't communicated clearly about this. I'd like to match the behavior of the `isdev` package here because that is as close to a standard that I know of.
```suggestion
      ["development", "dev", "", undefined].includes(process.env.NODE_ENV)
```